pyproject.toml

```diff
@@ -55,7 +55,8 @@ tests = [
 ]
 strict = [
     "pydantic==1.9.0",
-    "pymatgen==2022.7.8",
+    "pymatgen==2022.8.23",
+    "pymatgen-analysis-defects==2022.8.24",
```

Comment:
Please remove this! This will be an option dependency.

Reply:
I will.

Reply:
I updated the non-strict installation part as well. I hope this is fine.